projects/dforce/index.js

```diff
@@ -245,11 +246,13 @@ function getTVLByChain(chain) {
 
 function getLendingTvl(chain, borrowed){
   return sdk.util.sumChainTvls(allControllers[chain].map(controller =>
-    getCompoundV2Tvl(controller, chain, addr => `${chain}:${addr}`, undefined, undefined, borrowed, undefined, {
-      blacklistedTokens: excludeAlliTokens[chain].concat([USXs[chain] ?? ""]),
-      abis: {
+    getCompoundUsdTvl(controller, chain, cether = "0x5ACD75f21659a59fFaB9AEBAf350351a8bfaAbc0", borrowed,
```

Comment:
Emmmm, so how about does not assign a value for this parameter?

Reply:
@0xngmi Or what should I do to make it correct?

Reply:
@Skyge think this line could be `getCompoundUsdTvl(controller, chain, "0x5ACD75f21659a59fFaB9AEBAf350351a8bfaAbc0", borrowed,`  and remove `abis =` from next line.

issue is, now  `cether, abis` are used without being defined first and this throws an error while compilation

Reply:
@g1nt0ki I need to pass my own abi to make compatible with `compound.js`, cause there are some differences:
```
getAlliTokens (dForce) <-> getAllMarkets (Compound)
priceOracle (dForce)    <-> oracle (Compound)
```

So that is why I want to pass my own abi, if I can not make changes in `compound.js`, maybe I have got to use another way to do this without using `compound.js`

Reply:
getCompoundUsdTvl(controller, chain, "0x5ACD75f21659a59fFaB9AEBAf350351a8bfaAbc0", borrowed,
      {
        oracle: abi['oracle'],
      ...

is fine

------

    getCompoundUsdTvl(controller, chain, "0x5ACD75f21659a59fFaB9AEBAf350351a8bfaAbc0", borrowed,
      abis = {
        oracle: abi['oracle'],
      ...

would fail compilation since `abis` is not defined